src/components/RegistryInfoContent/RegistryInfoContent.tsx

```diff
@@ -11,86 +11,77 @@ import { default as Typography } from '../../muiComponents/Heading';
 import { CommandContainer } from './styles';
 import { Props, State } from './types';
 
-/* eslint react/prop-types:0 */
-function TabContainer({ children }): JSX.Element {
-  return (
-    <CommandContainer>
-      <Typography
-        className={css`
-          padding: 0;
-          min-height: 170;
-        `}
-        component="div">
-        {children}
-      </Typography>
-    </CommandContainer>
-  );
-}
+const RegistryInfoContent: React.FC<Props> = props => {
+  const [tabPosition, setTabPosition] = useState<State['tabPosition']>(0);
```

Comment:
```suggestion
  const [tabPosition, setTabPosition] = useState(0);
```

Reply:
as discussed I think it should be ok to be explicit here